docs/ProcessingPipeline.md

```diff
@@ -0,0 +1,62 @@
+# Processing Pipeline
```

Comment:
Note: Since this is a copy from `GettingStarted.md`, I suggest adding, as a comment, to both docs, that there's a second copy elsewhere, so that someone changing one or the other remembers to keep them exactly the same. Otherwise, they may start to drift.

Reply:
I've consolidated this back into `GettingStarted.md`, as we don't have easy support for exposing these new `docs/*md` files in our official docs [here](https://docs.datadoghq.com/tracing/setup_overview/setup/ruby/).
It is possible to have a multi-file tracer setup in our official docs, but it will require a bit of work with the documentations team to get it set up in first place.